src/applications/vaos/new-appointment/components/VAFacilityPage/getEligibilityMessage.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { ELIGIBILITY_REASONS } from '../../../utils/constants';
+import { aOrAn, lowerCase } from '../../../utils/formatters';
+import NewTabAnchor from '../../../components/NewTabAnchor';
+import FacilityAddress from '../../../components/FacilityAddress';
+
+export default function getEligibilityMessage({
+  typeOfCare,
+  eligibility,
+  facilityDetails,
+  includeFacilityContactInfo = false,
+}) {
+  let content = null;
+  let title = null;
+
+  const requestReason = eligibility.requestReasons[0];
+  const directReason = eligibility.directReasons[0];
+
+  if (
+    requestReason === ELIGIBILITY_REASONS.notSupported &&
```

Comment:
Do we need to check for the `notSupported` request reason in these if statements?

Reply:
I believe so, but it's arguable. If they're on this modal they something stopping them from doing direct scheduling or requests. We've always fallen back to the request reasons, since those in theory should be less restrictive. The change in this PR is to not do that when requests aren't supported, since if a facility has turned off requests but has direct scheduling on, their intent is that you direct schedule, and so we should show the direct schedule related reasons.

If we don't check `notSupported` here, then we'll always show these direct schedule reasons first, when maybe addressing the request reasons would let you at least make a request.

Reply:
Ah ok got it. It was confusing to me at first because the message content only had to do with the direct scheduling eligibility reason.